pom.xml

```diff
@@ -125,31 +125,25 @@ under the License.
 
   <properties>
     <javaVersion>8</javaVersion>
-    <minimalMavenBuildVersion>3.5.4</minimalMavenBuildVersion>
-
-    <sisuVersion>0.9.0.M2</sisuVersion>
-    <classWorldsVersion>2.7.0</classWorldsVersion>
-    <commonsCliVersion>1.5.0</commonsCliVersion>
-    <commonsIoVersion>2.11.0</commonsIoVersion>
-    <commonsLangVersion>3.12.0</commonsLangVersion>
+    <classWorldsVersion>2.8.0</classWorldsVersion>
+    <commonsCliVersion>1.7.0</commonsCliVersion>
+    <commonsIoVersion>2.16.1</commonsIoVersion>
+    <commonsLangVersion>3.14.0</commonsLangVersion>
     <junitVersion>4.13.2</junitVersion>
     <hamcrestVersion>2.2</hamcrestVersion>
     <mockitoVersion>4.11.0</mockitoVersion>
-    <!-- plexus 2.1.1 is broken, no new release exists yet (nor will exist) -->
-    <plexusVersion>2.1.0</plexusVersion>
-    <plexusInterpolationVersion>1.26</plexusInterpolationVersion>
-    <!-- Blocked by 3.4.0+ changes, see MNG-7710 -->
-    <plexusUtilsVersion>3.5.1</plexusUtilsVersion>
-    <guiceVersion>5.1.0</guiceVersion>
-    <guavaVersion>32.0.1-jre</guavaVersion>
-    <guavafailureaccessVersion>1.0.1</guavafailureaccessVersion>
+    <plexusVersion>2.2.0</plexusVersion>
+    <plexusInterpolationVersion>1.27</plexusInterpolationVersion>
+    <guiceVersion>6.0.0</guiceVersion>
+    <guavaVersion>33.2.0-jre</guavaVersion>
```

Comment:
But, this one should be ok (Maven does not use, it is runtime dep of guice) -- just to get rid of red flags